### PR TITLE
[ServiceFabric] Updated client enum values to readable strings

### PR DESCRIPTION
**Pull request by msjeffreychen**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@msjeffreychen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jefchen). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: lib/services/serviceFabric/lib/models/index.js [hunk 1]
```diff
@@ -134,37 +130,37 @@ exports.PartitionHealthReport = require('./partitionHealthReport');
...
-  'HealthEvaluation.13' : exports.ReplicaHealthEvaluation,
-  'HealthEvaluation.14' : exports.DeployedServicePackageHealthEvaluation,
-  'HealthEvaluation.15' : exports.DeployedApplicationHealthEvaluation,
-  'HealthEvaluation.16' : exports.ServiceHealthEvaluation,
-  'HealthEvaluation.17' : exports.NodeHealthEvaluation,
-  'HealthEvaluation.18' : exports.ApplicationHealthEvaluation,
-  'HealthEvaluation.19' : exports.DeltaNodesCheckHealthEvaluation,
-  'HealthEvaluation.20' : exports.UpgradeDomainDeltaNodesCheckHealthEvaluation,
-  'HealthEvaluation.21' : exports.ApplicationTypeHealthEvaluation,
+  'HealthEvaluation.EventHealthEvaluation' : exports.EventHealthEvaluation,
```
Comment (amarzavery) [inline lib/services/serviceFabric/lib/models/index.js]: What happened? Why did you change this?

Comment (msjeffreychen) [inline lib/services/serviceFabric/lib/models/index.js]: As discussed in the other pr, we are changing the server side enum.

**Comment by amarzavery** [comment]
@msjeffreychen - Have you generated from the latest spec changes? Generate using the version 0.16.0-Nightly20160317 as I have made some bug fixes in the code gen.

**Comment by msjeffreychen** [comment]
I have just generated using the latest generator. Thanks.
